client/src/ForgotPassword.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+//import { useHistory } from 'react-router-dom';
+//import jwt from 'jsonwebtoken';
+import SimpleSnackbar from './SimpleSnackbar';
+
+import type { Theme } from '@material-ui/core/styles';
+
+import EmailInput from './EmailInput';
+//import StyledLink from './StyledLink';
+//import { UserContext } from './providers';
+
+const useStyles = makeStyles((theme: Theme) => {
+  const xPadding = 12;
+  const yPadding = 6;
+  const yMargin = 8;
+
+  return {
+    paper: {
+      maxWidth: 821 - theme.spacing(xPadding),
+      maxHeight: 732 - theme.spacing(yPadding),
+      borderRadius: 10,
+      marginTop: theme.spacing(yMargin),
+      marginBottom: theme.spacing(yMargin),
+      paddingTop: theme.spacing(yPadding),
+      paddingBottom: theme.spacing(yPadding),
+      paddingLeft: theme.spacing(xPadding),
+      paddingRight: theme.spacing(xPadding),
+      margin: 'auto',
+    },
+    header: { fontWeight: 'bold', marginBottom: 68 },
+    button: {
+      borderRadius: 0,
+      height: 62,
+      textTransform: 'none',
+    },
+  };
+});
+
+interface UserLoginData {
+  email: string;
+}
+
+const initialFormData: UserLoginData = {
+  email: '',
+};
+
+interface Error {
+  type: '' | 'email';
+  message: string;
+}
+
+function ForgotPassword() {
+  const classes = useStyles();
+  //const history = useHistory();
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [error] = React.useState<Error | null>(null);
```

Comment:
let's not handle error state here
hopefully, down the road, the email input we create will validate and do that on its own

Reply:
I basically copied this from the login page. The PasswordInput.tsx is expecting an error props to be sent in.